data_make.py

```diff
@@ -0,0 +1,588 @@
+#!/usr/bin/python3
+# Generate 'utf8proc_data.c' by parsing the Unicode data files 'UnicodeData.txt' etc.
+# from the Unicode Character Database.  Tested with UCD 13.0.0 & 14.0.0-dev.
+# Usage: data_make.py [options…] [<data-dir>]
+
+import re, os, sys, fileinput, copy, datetime, getopt, platform
+from collections import defaultdict
+
+Me     = os.path.basename(sys.argv[0])
+Vers   = '2.7.0dev'		# ???
+DDir   = "./data"
+Targ   = "utf8proc_data.out.c"
+UVers  = None
+gVerb  = 0
+gFmt   = 0
+gCMap  = False
+gS1Byt = False
+gFix26 = False
+
+
+def Print (s):
+	print(s, file=sys.stderr)
+
+def Error (msg):
+	Print("%s: ERROR: %s." % (Me, msg))
+
+def TextFile (name):
+	return DDir+ "/" +name+ ".txt"
+
+def Sed (expr, file):
+	return os.popen("/usr/bin/sed -En '" +expr+ "' '" +TextFile(file)+ "'", 'r', 1)
```

Comment:
Python has its own regex support; it's a little odd to call out to `sed` etcetera